server/env.ts

```diff
@@ -601,6 +601,24 @@ export class Environment {
     this.AWS_S3_UPLOAD_MAX_SIZE
   );
 
+  /**
+   * Iframely url
+   */
+  @IsOptional()
+  @IsUrl({
+    require_tld: false,
+    allow_underscores: true,
+    protocols: ["http", "https"],
+  })
+  public IFRAMELY_URL = process.env.IFRAMELY_URL ?? "https://iframe.ly";
+
+  /**
+   * Iframely API key
+   */
+  @IsOptional()
+  @CannotUseWithout("IFRAMELY_URL")
+  public IFRAMELY_API_KEY = process.env.IFRAMELY_API_KEY ?? "";
```

Comment:
```suggestion
  public IFRAMELY_API_KEY = this.toOptionalString(process.env.IFRAMELY_API_KEY);
```

This _is_ optional, empty strings lead to bugs 😉